examples/bsc-example/src/app/bsc.smart.contract.examples.ts

```diff
@@ -0,0 +1,63 @@
+import { TransactionHash } from '@tatumio/api-client'
+import { TatumBscSDK } from '@tatumio/bsc'
+
+const bscSDK = TatumBscSDK({ apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab' })
+
+//https://apidoc.tatum.io/tag/BNB-Smart-Chain#operation/BscBlockchainSmartContractInvocation
+export async function bscSmartContractExample(): Promise<void> {
+  // if you don't already have a wallet, address and private key - generate them
+  // https://apidoc.tatum.io/tag/BNB-Smart-Chain#operation/BscGenerateWallet
+  const { mnemonic } = await bscSDK.wallet.generateWallet()
+  // https://apidoc.tatum.io/tag/BNB-Smart-Chain#operation/BscGenerateAddressPrivateKey
+  const fromPrivateKey = await bscSDK.wallet.generatePrivateKeyFromMnemonic(mnemonic, 0)
+
+  // FUND YOUR SENDER ACCOUNT WITH BNB FROM https://testnet.binance.org/faucet-smart
```

Comment:
you should print address with index 0 here as well so devs know where to fund assets from faucet